config/Dockerfiles/tests.d/libvirt/01_temp_general

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -xe
+
+# Verify basic provisioning for all supplied providers
+# distros.exclude: fedora26 centos7 fedora28
```

Comment:
I thought you were going to have it run against centos7? this won't run against anything will it?

Reply:
It will run against fedora27.